inlang/packages/paraglide/paraglide-js/src/compiler/compile-bundle.test.ts

```diff
@@ -62,8 +65,7 @@ export const blue_moon_bottle = (inputs, options = {}) => {
 	const locale = options.locale ?? getLocale()
 	trackMessageCall("blue_moon_bottle", locale)
 	if (locale === "en") return en.blue_moon_bottle(inputs)
-	if (locale === "en-US") return en_us2.blue_moon_bottle(inputs)
-	return "blue_moon_bottle"
+	return en_us2.blue_moon_bottle(inputs)
 };"`
 	);
 });
```

Comment:
Can you add a unit test for a bundle that has a missing message and therefore falls back to the id?

Reply:
Added